src/compose/ComposeMenu.js

```diff
@@ -80,6 +87,7 @@ class ComposeMenu extends PureComponent<Props> {
   };
 
   handleImagePicker = () => {
+    requestCameraPermission();
```

Comment:
This looks like it will launch the permission request dialog asynchronously, and then immediately start the image picker. What happens if the image picker launch completes (presumably with a failure) before the user does anything with the permission request dialog?

Reply:
I haven't tested that yet, should I put that in try/catch block?